### PR TITLE
whatsapp web version

### PR DESCRIPTION
**Pull request by ravindu01manoj**

### CODE REVIEW

**Comment by barkerpedro** [comment]
You can put the following link instead:
```typescript
const URL = 'https://web.whatsapp.com/check-update?version=1&platform=web'
```

**Comment by ravindu01manoj** [comment]
> You can put the following link instead:
> 
> ```ts
> const URL = 'https://web.whatsapp.com/check-update?version=1&platform=web'
> ```

okey :)

**Comment by adiwajshing** [comment]
Ah no, would like to have control over the version being specified by baileys. In the future, if there is a major update -- we don't want baileys to break just because we're fetching and running the latest version from WA Web.

I would recommend adding another function to fetch from WA Web, and keeping the original one as is

**Comment by Sew01RaviduManoj01KingAndQueen** [comment]
> Ah no, would like to have control over the version being specified by baileys. In the future, if there is a major update -- we don't want baileys to break just because we're fetching and running the latest version from WA Web.
> 
> I would recommend adding another function to fetch from WA Web, and keeping the original one as is

.

File: src/Utils/generics.ts [hunk 1]
```diff
@@ -254,6 +254,28 @@ export const fetchLatestBaileysVersion = async() => {
 	}
 }
 
+/**
+ * A utility that fetches the latest web version of whatsapp.
+ * Use to ensure your WA connection is always on the latest version
+ */
+export const fetchLatestWaWebVersion = async() => {
+	const URL = 'https://web.whatsapp.com/check-update?version=1&platform=web'
```
Comment (adiwajshing) [inline src/Utils/generics.ts:262]: should be `url` not screaming-case

Comment (ravindu01manoj) [reply src/Utils/generics.ts]: oh

**Pull request merged**